### PR TITLE
silence some warnings

### PR DESCRIPTION
**Pull request by grosser**
@bquorning 

tried to get them all, but some are hard to get (inherited classes have no instance variables defined) without adding extra checks ... so leaving them for now ...

### CODE REVIEW

**Comment by grosser** [comment]
@bquorning ... rbx is failing ... but that got solved in master

File: test/migrator_test.rb [hunk 1]
```diff
@@ -1,16 +1,6 @@
 # frozen_string_literal: true
 require File.expand_path('../helper', __FILE__)
 
-class CowardlyMigration < BaseMigration
```
Comment (bquorning) [inline test/migrator_test.rb]: Unused?

Comment (grosser) [inline test/migrator_test.rb]: duplicated

**Comment by grosser** [comment]
I can haz +1 ?

**Comment by bquorning** [comment]
:+1:

**Pull request merged**